pkg/controllerutils/predicate/predicate.go

```diff
@@ -215,12 +214,3 @@ func GetExtensionLastOperation(obj client.Object) *gardencorev1beta1.LastOperati
 
 	return acc.GetExtensionStatus().GetLastOperation()
 }
-
-// SeedNamePredicate returns a predicate which returns true for objects that are being migrated to a different
-// seed cluster.
-func SeedNamePredicate(seedName string, getSeedNamesFromObject func(client.Object) (*string, *string)) predicate.Predicate {
```

Comment:
Did you consider whether we should deprecate this method first, since it's an exported utility function. It doesn't seem to be used in well-known extensions. But at the very least I would add a release note for it as a breaking change.